### DISCUSSION TITLE
Unable to style an active navbar item

### DISCUSSION THREAD

**Discussion by akvrmvlakis** [bootscore/bootscore#665] [Q&A]
### Describe the bug

I am unable to style an active menu item that has children. I have tried everything and searched all over for Bootscore similar issues but was not able to find anything. I think this is an issue related to Bootstrap and  Nav walker because I've seen others face the same issue with other Bootstrap-based themes. I would appreciate a solution if anyone has found a way around this. Many thanks in advance.

### Steps to reproduce

Inspect a navbar item that has children. The particular class in question is: `a.nav-link.dropdown-toggle.show` and although you can change it's color while inspecting and it seems that it's working when you add the change to _bscore_custom.scss and save it seems to have no effect.

### Screenshots and Additional Info

![bootscore-screenshot](https://github.com/bootscore/bootscore/assets/25525915/a8520271-8859-4b63-9522-24c65cd0c181)

### Website link

etsak.cloud

**Comment by crftwrk** [comment]
> ... and it seems that it's working when you add the change to _bscore_custom.scss and save it seems to have no effect.

It seems that your compiler stops working, enable dev mode to force recompile https://bootscore.me/documentation/theme/bootstrap-css-sass/#Developer_mode.

When dev mode is enabled, your changes should be as expected and you have two options to style the nav-links.

Use variables in `_bscore_variables.scss`:

```scss
$navbar-light-color:        #339900;
$navbar-light-hover-color:  #FF0000;
$navbar-light-active-color: #00FF00;
```

Or override values in `_bscore_custom.scss`:

```scss
.navbar-nav {
  .nav-link {
    color: #339900;

    &:hover,
    &:focus {
      color: #FF0000;
    }

    &.active,
    &.show {
      color: #00FF00;
    }
  }
}
```

Both leads into the same result, but first option is better.

Solved?

**Comment by akvrmvlakis** [comment]
The issue is solved. Thank you very much for looking into this and I apologize for calling in the big guns for something of such low importance I had honestly overlooked it and couldn't find a solution. I wish you happy holidays and thank you for your work on this excellent theme.
